tests/integration/ha_tests/test_ha.py

```diff
@@ -125,6 +125,7 @@ async def test_kill_db_process(ops_test: OpsTest, continuous_writes):
     stop_writes_action = await application_unit.run_action("stop-continuous-writes")
     await stop_writes_action.wait()
     total_expected_writes = int(stop_writes_action.results["writes"])
+    assert total_expected_writes > 0, "error while getting total writes."
```

Comment:
Make sure that the test charm actually wrote something and didn't simply error out.